internal/viridian/cluster.go

```diff
@@ -65,34 +68,40 @@ func (a API) StopCluster(ctx context.Context, idOrName string) error {
 	if err != nil {
 		return err
 	}
-	ok, err := doPost[[]byte, bool](ctx, fmt.Sprintf("/cluster/%s/stop", c.ID), a.Token(), nil)
+	ok, err := WithRetry(ctx, a, func() (any, error) {
+		return doPost[[]byte, bool](ctx, fmt.Sprintf("/cluster/%s/stop", c.ID), a, nil)
+	})
 	if err != nil {
 		return fmt.Errorf("stopping cluster: %w", err)
 	}
-	if !ok {
+	if !ok.(bool) {
 		return errors.New("could not stop the cluster")
 	}
 	return nil
 }
 
 func (a API) ListClusters(ctx context.Context) ([]Cluster, error) {
-	csw, err := doGet[Wrapper[[]Cluster]](ctx, "/cluster", a.Token())
+	csw, err := WithRetry(ctx, a, func() (any, error) {
+		return doGet[Wrapper[[]Cluster]](ctx, "/cluster", a)
+	})
 	if err != nil {
 		return nil, fmt.Errorf("listing clusters: %w", err)
 	}
-	return csw.Content, nil
+	return csw.(Wrapper[[]Cluster]).Content, nil
 }
 
 func (a API) ResumeCluster(ctx context.Context, idOrName string) error {
 	c, err := a.FindCluster(ctx, idOrName)
 	if err != nil {
 		return err
 	}
-	ok, err := doPost[[]byte, bool](ctx, fmt.Sprintf("/cluster/%s/resume", c.ID), a.Token(), nil)
+	ok, err := WithRetry(ctx, a, func() (any, error) {
```

Comment:
Same comment with other `WithRetry` code.

Reply:
Fixed in: https://github.com/hazelcast/hazelcast-commandline-client/pull/300/commits/d3d3b8eb6da57e1de512a87e7ebd5597a1c31c6f